app/src/main/java/ch/epfl/sweng/swenggolf/main/MainMenuActivity.java

```diff
@@ -0,0 +1,113 @@
+package ch.epfl.sweng.swenggolf.main;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.design.widget.NavigationView;
+import android.support.v7.app.AppCompatActivity;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+import ch.epfl.sweng.swenggolf.database.FakeUserDatabase;
+import ch.epfl.sweng.swenggolf.database.FirebaseAccount;
+import ch.epfl.sweng.swenggolf.offer.CreateOfferActivity;
+import ch.epfl.sweng.swenggolf.offer.ListOfferActivity;
+import ch.epfl.sweng.swenggolf.profile.ProfileActivity;
+import ch.epfl.sweng.swenggolf.R;
+
+public class MainMenuActivity extends AppCompatActivity {
+    private FirebaseAccount account;
+    private View nav;
+
+    @Override
+    protected void onCreate(Bundle savedInstances) {
+        super.onCreate(savedInstances);
+        setContentView(R.layout.activity_main_menu);
+        android.support.v7.widget.Toolbar tb = findViewById(R.id.toolbar);
+        nav = ((NavigationView)(this.findViewById(R.id.drawer))).getHeaderView(0);
+        setSupportActionBar(tb);
+        setUserDisplay();
```

Comment:
setUserDisplay() returns a boolean but you don't check it ? What happens if it is false ?

Reply:
Nothing it just says if it could fill in the textViews with user data or left with default fields